.github/workflows/pull-request.yml

```diff
@@ -99,7 +99,7 @@ jobs:
           cache: 'npm'
       - uses: ./.github/actions/before-install
       - run: npm ci
-      - run: npm run affected:test -- --browsers=ChromeHeadless --watch=false $TEST_OPTS --base=origin/develop
+      - run: npm run affected:test -- --browsers=ChromeHeadless --watch=false $TEST_OPTS --base=origin/develop --exclude='content-ce'
```

Comment:
This should not be probably a fix for that problem. What in case when somebody will add any test there? Then these tests will be excluded and never ran.